src/server/context.ts

```diff
@@ -56,6 +56,11 @@ function isObject(value: unknown) {
     value !== null;
 }
 
+function isDevMode() {
+  // Env var is only set in prod (on Deploy).
+  return typeof Deno.env.get("DENO_DEPLOYMENT_ID") !== "string";
```

Comment:
```suggestion
  return Deno.env.get("DENO_DEPLOYMENT_ID") === undefined;
```
Nit. Reference: https://deno.land/api@v1.34.1?s=Deno.Env#method_get_0